open_seq2seq/models/text2speech.py

```diff
@@ -0,0 +1,383 @@
+# Copyright (c) 2018 NVIDIA Corporation
+
+from __future__ import absolute_import, division, print_function
+from __future__ import unicode_literals
+from six.moves import range
+
+import pandas as pd
+import tensorflow as tf
+import numpy as np
+
+import matplotlib as mpl
```

Comment:
Can you add matplotlib into requirements.txt? And any additional dependencies that your code requires.